src/migrations/20230803061359-change-request-optional-feature.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+            ALTER TABLE change_request_events ALTER COLUMN feature DROP NOT NULL;
+        `,
+        callback,
+    );
+};
+
+exports.down = function (db, callback) {
+    db.runSql(
+        `
+            UPDATE change_request_events SET feature = 'unknown' WHERE feature IS NULL;
```

Comment:
This will not work, because it is invalid foreign key.

Reply:
good point, what options do we have? delete those?

Reply:
Yeah, not ideal, but acceptable.

Reply:
FYI, there is no ideal solution. We kind of want to retain all user data, but it will be super hard anyways, if they do down migrations.